packages/core/src/storage/migration/__tests__/__snapshots__/0.1.test.ts.snap

```diff
@@ -2893,6 +2893,2796 @@ Object {
       "threadId": "e9aeea8f-2c7a-4fd0-9353-f8b5b76094e7",
     },
   },
+  "STORAGE_VERSION_RECORD_ID": Object {
```

Comment:
I'm not sure why there are so much changes? It should only add a few new keys.....